app/models/mixins/kubernetes_provider_mixin.rb

```diff
@@ -1,6 +1,5 @@
 module KubernetesProviderMixin
   extend ActiveSupport::Concern
-
```

Comment:
why is this change part of this PR? there are no other changes in the file related to the PR